packages/render-html/src/hooks/useTRenderEngine.ts

```diff
@@ -1,32 +1,44 @@
 import { useMemo } from 'react';
+import { Platform } from 'react-native';
 import { TRenderEngineConfig } from '../shared-types';
 import buildTREFromConfig from '../helpers/buildTREFromConfig';
 import useProfiler from './useProfiler';
+import defaultSystemFonts from '../defaultSystemFonts';
+
+/**
+ * Default fallback font for special keys such as 'sans-serif', 'monospace',
+ * 'serif', based on current platform.
+ */
+export const defaultFallbackFonts = {
+  'sans-serif': Platform.select({ ios: 'system', default: 'sans-serif' }),
+  monospace: Platform.select({ ios: 'Menlo', default: 'monospace' }),
+  serif: Platform.select({ ios: 'Times New Roman', default: 'serif' })
+};
 
 /**
  * @internal
  */
 export default function useTRenderEngine({
   allowedStyles,
-  baseStyle,
-  classesStyles,
-  customHTMLElementModels,
+  baseStyle = { fontSize: 14 },
+  classesStyles = {},
+  customHTMLElementModels = {},
   dangerouslyDisableHoisting,
   dangerouslyDisableWhitespaceCollapsing,
   domVisitors,
-  emSize,
-  enableCSSInlineProcessing,
-  enableUserAgentStyles,
-  fallbackFonts,
-  htmlParserOptions,
+  emSize = 14,
+  enableCSSInlineProcessing = true,
+  enableUserAgentStyles = true,
+  fallbackFonts = defaultFallbackFonts,
+  htmlParserOptions = { decodeEntities: true },
   idsStyles,
   ignoreDomNode,
-  ignoredDomTags,
-  ignoredStyles,
+  ignoredDomTags = [],
+  ignoredStyles = [],
   selectDomRoot,
   setMarkersForTNode,
-  systemFonts,
-  tagsStyles
+  systemFonts = defaultSystemFonts,
+  tagsStyles = {}
 }: TRenderEngineConfig) {
```

Comment:
These changes will break memoization below. Consider extracting default props to a referentially stable immutable object